addons/mail/static/src/core/web/activity_menu.js

```diff
@@ -85,6 +85,11 @@ export class ActivityMenu extends Component {
             }
         );
     }
+
+    openMyActivities() {
+        this.dropdown.close();
+        this.action.doAction("mail.mail_activity_action_my", {clearBreadcrumbs: true});
```

Comment:
(minor) applying (automatic) formatter would result in adding spaces around brackets of inline object